docs/rules/max-depth.md

```diff
@@ -1,13 +1,12 @@
-# Limit Maximum Depth (max-depth)
+# enforce a maximum depth that blocks can be nested (max-depth)
 
 The `max-depth` rule allows you to specify the maximum depth blocks can be nested.
 
 ```js
 function foo() {
-  for (;;) { // Nested 1 deep.
-    if (true) { // Nested 2 deep.
-      if (true) { // Nested 3 deep.
-
+  for (;;) { // Nested 1 deep
```

Comment:
It looks like the example code in this doc was indented 2 spaces instead of 4 spaces, which is the convention for ESLint code. Can you fix that? Thanks.